### PR TITLE
Typography settings

### PR DESCRIPTION
**Pull request by ascreven-gsa**
Addresses refactors in IAE-38190 and IAE-38191

### Cleanup ###
- removed sam-content preview page
- removed all instances of `usa-prose` class
- removed unused config and readme files
- updated typography settings page from USWDS 2.2 to 2.6
- updated typography settings based on PI-27 Master Sketch and measure based on guidance from Christy

### Typography Demos ###
- updated fields typography table to match Sketch file
- fixed heading link typography table to match Sketch file
- corrected heading link `a` tag
- updated link example to use reusable link component
- updated typesetting paragraph measure to reflect SDS setting
- removed table demo from typesetting since it's now its own component
- most changes to typesetting page are just spacing corrections

### Components ###
- updated icon collators for new spacing
- updated card title top margin for use with new typography settings

### SCSS ###
- refactored hardcoded font-weight
- removed text styling where settings are used
- added list item line height since different from default
- refactored typography header to avoid unintended consequences of using @extends
- *HTML in page example needs to be refactored. Designers are aware that it may look a little different until changes are made to its structure*

Checked in:
- Chrome
- Firefox
- Safari
- IE
- Edge

### CODE REVIEW

File: src/components/01-branding/02-typography/fields/fields.njk [hunk 1]
```diff
@@ -11,57 +11,57 @@
...
+      <td>Weight: Semi-Bold / Size: 1.13rem / Line Height: 1.1 / Color: Base-Dark (#454540)</td>
+    </tr>
     <tr>
       <td><element class="sds-field">Field Label</element></td>
       <th scope="row">&ltelement class="sds-field">&lt/element></th>
-      <td>Weight: Regular / Size: 0.93rem / Line Height: 1.5rem / Color: Base-Dark (#454540)</td>
+      <td>Weight: Regular / Size: 0.93rem / Line Height: 1.3 / Color: Base-Dark (#454540)</td>
     </tr>
     <tr>
       <td><element class="sds-field--large">Large Field Label</element></td>
```
Comment (AlejandroColinaGSA) [inline src/components/01-branding/02-typography/fields/fields.njk:27]: There is this non-standard element tag throughout, please use standard HTML replacement

Comment (ascreven-gsa) [reply src/components/01-branding/02-typography/fields/fields.njk:27]: `td` is a standard `table` element for table data. The ticket wasn't meant to refactor the fields.njk file. Rather I just corrected the displayed line height  to be valid line height value and updated the number based the Sketch file and guidance from the designers.

Comment (AlejandroColinaGSA) [reply src/components/01-branding/02-typography/fields/fields.njk:27]: `<element class="sds-field">` is not proper HTML.

Comment (ascreven-gsa) [reply src/components/01-branding/02-typography/fields/fields.njk:27]: Please review the updates as the ticket was not a typography demo refactor. Please refer to the PR summary for a full list of the branch changes.

Comment (AlejandroColinaGSA) [reply src/components/01-branding/02-typography/fields/fields.njk:27]: Correct, I am not calling for a refactor, just quality and best practices.

Comment (AlejandroColinaGSA) [reply src/components/01-branding/02-typography/fields/fields.njk:27]: I'll close this, please insure that a backlog ticket/entry is created to address this

File: src/components/01-branding/02-typography/list/list.njk [hunk 2]
```diff
@@ -1,4 +1,5 @@
 {% if title %}
```
[the remark this answers is not in the document]
Comment (ascreven-gsa) [reply src/components/01-branding/02-typography/list/list.njk:1]: The only thing updated was a class but this nested component has existed for some time and is used in the lists in the landing page cards.

[the remark this answers is not in the document]
Comment (ascreven-gsa) [reply src/components/01-branding/02-typography/list/list.njk:1]: > .sds-field__small is not working. Please see this the example page

I am unsure what you mean. The class is `sds-field--small` not `sds-field__small` as shown in the Fields previews. Additionally, this class wasn't updated in the changes but the line-height value was corrected after talking to Alanna.

**Pull request merged**